pycloudlib/openstack/instance.py

```diff
@@ -39,6 +41,7 @@ def __init__(self, key_pair, instance_id, connection=None):
         if self.floating_ip is None:
             self.floating_ip = self._create_and_attach_floating_id()
             self.delete_floating_ip = True
+        self.added_local_ports = []
```

Comment:
Do we eventually want visibility to the ports attached to the running instance via `compute.server_interfaces` so that other pre-existing ports could be removed via pycloudlib (in the event of an instance created and referenced via 
 pycloudlib's `Openstack.get_instance`?

Reply:
I'm not entirely sure what you're asking here.